js/skin.js

```diff
@@ -241,7 +241,7 @@ var Skin = React.createClass({
               icon="discovery">
               <DiscoveryPanel
                 {...this.props}
-                videosPerPage={{xs:2, sm:4, md:6, lg:8}}
+                videosPerPage={this.state.config.nym && this.state.config.nym.discoveryVideosPerSlide || {xs:2, sm:4, md:6, lg:8}}
```

Comment:
Maybe something like `videosPerPage = _.assign({xs:2, sm:4, md:6, lg:8}, this.state.config.nym ? this.state.config.nym.discoveryVideosPerSlide : {} })`